submissions/asaMitaka/FriendsApp/script.js

```diff
@@ -0,0 +1,177 @@
+const url = 'https://randomuser.me/api/?results=12';
+
+let aToZLast = document.querySelector('.sortedAtoZLast');
+let zToALast = document.querySelector('.sortedZtoALast');
+let atoZFirst = document.querySelector('.sortedAtoZFirst');
+let ztoAFirst = document.querySelector('.sortedZtoAFirst');
+let content = document.querySelector('.articleBlock');
+let reset = document.querySelector('.reset');
+let male = document.querySelector('#male');
+let woman = document.querySelector('#female');
+let all = document.querySelector('#all');
+let search = document.querySelector('#searchBar');
+let minNumber = document.querySelector('#minNumber');
+let maxNumber = document.querySelector('#maxNumber');
+
+let basicArr1 = [];
+let resultArr = [];
+
+async function getResponse(url) {
+    let response = await fetch(url);
+    let friendsArr = await response.json();
+
+    saveUsers(friendsArr.results, resultArr);
+
+    renderAllItemsToPage(resultArr);
+}
+
+
+function saveUsers(arr, resArr) {
+    for (let i = 0; i < arr.length; i++) {
+        resArr.push(arr[i]);
+    }
+    basicArr1 = resultArr.slice();
+    // basicArr2 = JSON.parse(JSON.stringify(resultArr));
+}
+
+function renderItem(item) {
```

Comment:
Misleading name, this function doesn't `render` anything, it returns an HTML code for an element.